Cpp/Library/Control/Controller.hh

```diff
@@ -76,43 +88,43 @@ namespace Nymph
             MEMVAR_REF_MUTABLE( std::condition_variable, CondVarContinue );
```

Comment:
Should the mutex and condition variables really be public members?

Reply:
These variables are not actually public.  That macro expands to something like:
```
public:
    std::condition_variable& CondVarContinue() {...}
    const std::condition_variable& CondVarContinue() const {...}
protected:
    std::condition_variable fCondVarContinue;
```
(plus `mutable` as needed in this case, which I left out for clarity)

These macros are always explicit about the access specifiers for their member variables and access functions.  One does have to be careful to always explicitly specify the access specifier that you want for any code coming _after_ the use of a macro.